docs/source/tutorial/visualise_pipeline.md

```diff
@@ -97,11 +97,13 @@ kedro viz --load-file my_shareable_pipeline.json
 This will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
 
 
-## Visualise Plotly charts in Kedro-Viz
+## Visualising charts in Kedro-Viz
 
-[Plotly](https://plotly.com/python/) is a free and open source Python library that allows you to make interactive, publication-quality graphs. With the Plotly integration on Kedro-Viz, you can output your interactive charts as part of your pipeline visualisation.
+Kedro-Viz aims to help users communicate different aspects of their workflow through an interactive flowchart. With the Plotly and Matplotlib integrations, we take one step further in this direction to allow our users to share their data insights effectively while exploring the pipeline.
```

Comment:
Outside the scope of your change I know, but I'm personally not a big fan of this paragraph. It feels too vague and marketing-ese to me. Feel free to change it as you please or leave it as it is depending on what you think.

Reply:
@rashidakanchwala any thoughts on how to restructure this?

Reply:
No idea, i personally liked this because it excites the user (by technically advertising the feature) but i agree with Antony too.

i will let @yetudada comment.

Reply:
I wouldn't remove it completely - but I would suggest a modification. We describe Kedro-Viz here and also at the top of the page so I would remove the first sentence because we don't need to do it twice. 

So it could be something like: "Kedro-Viz supports [Plotly](https://plotly.com/python/) and [Matplotlib](https://matplotlib.org/) integrations so that you can share your data insights while exploring your pipeline."